### ISSUE TITLE
Some context options cannot be changed if the assigned hotkey includes a modifier key 

### ISSUE THREAD

**Issue by Gasparoken** [aseprite/aseprite#3784]
Some options available in the context bar cannot be changed via keyboard shortcut if:
- The shortcut includes a modifier as "Space", "Ctrl", "Alt" and
- The cursor is on the canvas viewport.

Example: Pixel Perfect switch
By default, "Space", "Ctrl", "Alt" switches the context bar, so the Pixel Perfect switch will not be accessible.

Steps to reproduce it:
- Go to _Edit > Keyboard Shorcuts > Switch Pixel Perfect Mode_ set "Space + X".
- "Space + X" when the cursor is on the workspace will not change the Pixel Perfect switch.

This issue was created from observations of: https://github.com/aseprite/aseprite/issues/3773#issuecomment-1491092452

Aseprite version: 1.3rc1
OS version: Windows and MacOS

**Comment by ElderlyGlek** [comment]
Hi, @Gasparoken. I would like to ask if this bug will be fixed soon? I would also like to suggest adding a "Same in all Tools" parameter for "Pixel-perfect" and "Contiguous", as I noticed that it is very inconvenient to switch "Pixel-perfect" and "Contiguous" for each tool separately.

**Comment by ElderlyGlek** [comment]
@Gasparoken, sorry to bother you, but I have another question. Why does the "Spray" tool need "Pixel-perfect"? Personally, I don't notice any difference between "Pixel-perfect" on and off.

**Comment by Gasparoken** [comment]
Yeah, you are right. I think the Pixel Perfect checkbox in the Spray tool is irrelevant.
